src/oss-characteristics/CharacteristicTool.cs

```diff
@@ -73,11 +74,15 @@ public static IEnumerable<Example> Examples
             public FailureLevel SarifLevel { get; set; } = FailureLevel.Note;
         }
 
+        public CharacteristicTool(ProjectManagerFactory projectManagerFactory) : base(projectManagerFactory)
+        {
+        }
+
         public CharacteristicTool(IHttpClientFactory httpClientFactory) : base(httpClientFactory)
```

Comment:
Remove

Reply:
I think we should be removing all the IHttpClient based ctors.

Reply:
Or like below it should make a projectmanagerfactory with that httpclient to maintain compat.